test/metabase/api/persist_test.clj

```diff
@@ -4,7 +4,9 @@
    [metabase.models.database :refer [Database]]
    [metabase.task.persist-refresh :as task.persist-refresh]
    [metabase.test :as mt]
-   [metabase.test.fixtures :as fixtures]))
+   [metabase.test.fixtures :as fixtures]
+   [metabase.util :as u]
+   [toucan2.tools.with-temp :as t2.with-temp]))
```

Comment:
```suggestion
))
```

need to remove this once you switched to using `mt/with-temp` per the suggestion below